### ISSUE TITLE
getSubmission() must be an instance of verbb\formie\elements\Submission

### ISSUE THREAD

**Issue by rtrudel** [verbb/formie#325]
**Description**

When trying to open a notification (in Sent Notifications), this error occurs:

`Return value of verbb\formie\elements\SentNotification::getSubmission() must be an instance of verbb\formie\elements\Submission, null returned`

**Steps to reproduce**
1. Go to Sent notifications
2. Click one 

### Form settings
Please pick from the options below.

- Multi-page form:  No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.14
- Craft version: 3.6
- Multi-site: No

**Additional context**
Using in devMode

**Comment by rtrudel** [comment]
Just want to add: thumbs up for this incredible plugin! Love it!

**Comment by engram-design** [comment]
I can't replicate this specifically, but I would assume the submission has been deleted? Let me know if not, as it might be something else.

Should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.15](https://github.com/verbb/formie/releases/tag/1.3.15)
